pkg/pipelinerunmetrics/metrics.go

```diff
@@ -213,6 +213,7 @@ func viewRegister(cfg *config.Metrics) error {
 		Description: runningPRs.Description(),
 		Measure:     runningPRs,
 		Aggregation: view.LastValue(),
+		TagKeys:     append([]tag.Key{namespaceTag}, prunTag...),
```

Comment:
This should be configurable based on some key in config map.

Reply:
I believe we should always add the namespace tag.
currently prunTag is being used at two places and both with namespaces. what about adding the namespace in pruntag only ?

Reply:
Please see this issue: https://github.com/tektoncd/pipeline/issues/3171

Reply:
Also this: https://github.com/knative/pkg/pull/1464

Reply:
https://github.com/knative/pkg/pull/2931